qutebrowser/browser/commands.py

```diff
@@ -238,6 +247,23 @@ def tab_close(self, left=False, right=False, opposite=False, count=None):
             self._tabbed_browser.close_tab(tab)
             tabbar.setSelectionBehaviorOnRemove(old_selection_behavior)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window', name='tab-pin')
+    @cmdutils.argument('index')
+    @cmdutils.argument('count', count=True)
+    def tab_pin(self, index=None, count=None):
+        tab = self._cntwidget(count)
+        if tab is None:
+            return
+
+        tab.pin = not tab.pin
+
+        if tab.pin is True:
```

Comment:
When checking for booleans, just do `if tab.pin:` (or `if tab.data.pinned:` with the changes above) - see the last point in ["Programming Recommendations"](http://legacy.python.org/dev/peps/pep-0008/#programming-recommendations) in the pep8 style guide.